Tasks/NuGetInstallerV0/package.json

```diff
@@ -19,7 +19,6 @@
   "dependencies": {
     "packaging-common": "file:../../_build/Tasks/Common/packaging-common-1.0.1.tgz",
     "utility-common": "file:../../_build/Tasks/Common/utility-common-1.0.2.tgz",
-    "vsts-task-lib": "2.6.0",
```

Comment:
do we need to add 'azure-pipelines-task-lib' here?

Reply:
no, it gets pulled in with utility-common.